frontend/src/pages/AboutUs/AboutUs.tsx

```diff
@@ -0,0 +1,151 @@
+import {
+  Container,
+  Typography,
+  ThemeProvider,
+  createTheme,
+  Box,
+} from '@mui/material';
+
+const theme = createTheme({
+  typography: {
+    fontFamily: 'Arial, sans-serif',
+    h2: {
+      color: '#000000',
+    },
+    body1: {
+      color: '#213343',
+    },
+  },
+});
+
+const styles = {
+  root: {
+    backgroundColor: '#ffffff',
+    paddingBottom: '4rem',
+    marginTop: '3rem',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  section: {
+    padding: '4rem 0',
+    borderBottom: '1px solid #e0e0e0',
+    width: '100%',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    textAlign: 'center',
+  },
+  image: {
+    width: '23rem',
+    height: '13rem',
+    borderRadius: '8px',
+    marginBottom: '2rem',
```

Comment:
Let's keep the image width `32rem` for larger screens only. Also, please don't set the height because it is making the image stretched.